src/main/java/shedar/mods/ic2/nuclearcontrol/items/ItemPanelMemoryCard.java

```diff
@@ -0,0 +1,89 @@
+package shedar.mods.ic2.nuclearcontrol.items;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.world.World;
+import net.minecraftforge.common.util.Constants;
+import shedar.mods.ic2.nuclearcontrol.IC2NuclearControl;
+import shedar.mods.ic2.nuclearcontrol.tileentities.TileEntityInfoPanel;
+import shedar.mods.ic2.nuclearcontrol.utils.TextureResolver;
+
+import java.util.List;
+
+public class ItemPanelMemoryCard extends Item {
+
+	public ItemPanelMemoryCard() {
+		super();
+		setMaxStackSize(1);
+		setCreativeTab(IC2NuclearControl.tabIC2NC);
+		setTextureName(TextureResolver.getItemTexture("panelMemoryCard"));
+	}
+
+	@Override
+	@SideOnly(Side.CLIENT)
+	public void addInformation(ItemStack itemStack, EntityPlayer player,
+							   List info, boolean advanced) {
+		NBTTagCompound nbt = itemStack.getTagCompound();
+		if (nbt != null) {
+			// Count saved settings for existing cards
+			int cardsSaved = 0;
+			for (int i = 0; i < 4; i++) {
+				String key = "dSettings";
+				if (i > 0) key += i;
+				if (nbt.hasKey(key)) {
+					NBTTagList settings = nbt.getTagList(key, Constants.NBT.TAG_COMPOUND);
+					if (settings.tagCount() > 0)
+						cardsSaved++;
+				}
+			}
+
+			if (nbt.hasKey("rotateHor"))
+				info.add("Contains panel metadata");
+			if (cardsSaved > 0)
+				info.add("Contains settings for " + cardsSaved + " card(s)");
+		}
+	}
+
+	@Override
+	public boolean onItemUseFirst(ItemStack itemStack, EntityPlayer player,
+								  World world, int x, int y, int z, int side,
+								  float hitX, float hitY, float hitZ) {
+		boolean isServer = player instanceof EntityPlayerMP;
+		if (!isServer)
+			return false;
```

Comment:
Make sure to return true here if the server side returns true, otherwise desyncs can happen and glitchy looking behaviour with the wrong animations